core/jest.config.js

```diff
@@ -5,14 +5,18 @@ module.exports = {
   testTimeout: helper.defaultTime,
   maxWorkers: "50%",
   setupFiles: ["<rootDir>/jest.setup.js"],
-  transform: {
-    "^.+\\.ts?$": "ts-jest",
-  },
+  preset: "ts-jest",
   testPathIgnorePatterns: [
     "<rootDir>/__tests__/utils",
     "<rootDir>/__tests__/fixtures",
     "<rootDir>/src/.*/*.ts",
     "<rootDir>/dist/.*/*.ts",
     "<rootDir>/dist/.*/*.js",
   ],
+  globals: {
+    "ts-jest": {
+      transpileOnly: true,
+      isolatedModules: true,
```

Comment:
Wondering if we want to add this to the compiler or to other suites? 
https://github.com/kulshekhar/ts-jest/issues/1967#issuecomment-1017510990